R/Rules-class.R

```diff
@@ -139,13 +169,101 @@ validObject(.NextBestNCRM())
 ##' @keywords methods
 NextBestNCRM <- function(target,
                          overdose,
-                         maxOverdoseProb)
-{
-    .NextBestNCRM(target=target,
-                  overdose=overdose,
-                  maxOverdoseProb=maxOverdoseProb)
+                         maxOverdoseProb) {
+  .NextBestNCRM(
+    target = target,
+    overdose = overdose,
+    maxOverdoseProb = maxOverdoseProb
+  )
 }
 
+##' Next best dose based on NCRM rule and loss function
+##'
+##' This class is similar to NCRM class, the only difference is the addition of
+##' loss function. As in NCRM rule, first admissible doses are found, which are those
+##' with probability to fall in \code{overdose} category being below
+##' \code{maxOverdoseProb}. Next, within the admissible doses, the loss function is
+##' calculated, i.e. \code{losses} %*% \code{target}. Finally, the corresponding
+##' dose with lowest loss function (Bayes risk) is recommended for the next dose.
+##' @slot target the target toxicity interval (limits included)
+##' @slot overdose the overdose toxicity interval (lower limit excluded, upper limit included)
+##' @slot maxOverdoseProb maximum overdose probability that is allowed
+##' @slot losses loss function
```

Comment:
I think here it would be good to add additional details. As I see from line 209 this is a  vector of losses for some categories or so, so not really a function in the syntax sense.

Reply:
More documentation is added as suggested (exactly it is a numeric vector actually)